rmgpy/molecule/molecule.py

```diff
@@ -1863,62 +1863,74 @@ def to_single_bonds(self, raise_atomtype_exception=True):
         new_mol.update_atomtypes(raise_exception=raise_atomtype_exception)
         return new_mol
 
-    def to_inchi(self):
+    def to_inchi(self, backend='rdkit-first'):
         """
         Convert a molecular structure to an InChI string. Uses
         `RDKit <http://rdkit.org/>`_ to perform the conversion.
         Perceives aromaticity.
-        
+
         or
-        
+
         Convert a molecular structure to an InChI string. Uses
         `OpenBabel <http://openbabel.org/>`_ to perform the conversion.
+
+        Available options for InChI backend: 'rdkit-first' (default),
+        'try-all', 'rdkit', or 'openbabel'.
```

Comment:
Could you add an explanation in the docstring on what packages `try-all` uses and their sequence?

Reply:
@xiaoruiDong can you do this?